beck-online.js

```diff
@@ -118,6 +115,57 @@ function authorRemoveTitlesEtc(authorStr) {
 	return ZU.trimInternal(ZU.trimInternal(authorStr).replace(authorRegEx, ""));
 }
 
+function scrapeKommentar(doc, url) {
+	var item = new Zotero.Item("encyclopediaArticle");
+	
+	item.title = ZU.xpathText(doc, '//div[@class="dk2"]//span[@class="ueber"]');
+	
+	var authorText = ZU.xpathText(doc, '//div[@class="dk2"]//span[@class="autor"]');
+	if (authorText) {
+		var authors = authorText.split("/");
+		for (var i=0; i<authors.length; i++) {
+			item.creators.push(ZU.cleanAuthor(authors[i], 'author', false));
+		}
+	}
+	
+	//e.g. a) Beck'scher Online-Kommentar BGB, Bamberger/Roth
+	//e.g. b) Langenbucher/Bliesener/Spindler, Bankrechts-Kommentar
+	var citationFirst = ZU.xpathText(doc, '//div[@class="dk2"]//span[@class="citation"]/text()[following-sibling::br and not(preceding-sibling::br)]', null, ' ');//e.g. Beck'scher Online-Kommentar BGB, Bamberger/Roth
+	var pos = citationFirst.lastIndexOf(",");
+	if (pos > 0) {
+		item.publicationTitle = ZU.trimInternal(citationFirst.substr(0, pos));
+		var editorString = citationFirst.substr(pos+1);
+		
+		if (editorString.indexOf("/") == -1 && item.publicationTitle.indexOf("/") > 0) {
+			var temp = item.publicationTitle;
+			item.publicationTitle = editorString;
+			editorString = temp;
+		}
+		editorString = editorString.replace(/, /g, '');
+		
+		var editors = editorString.trim().split("/");
+		for (var i=0; i<editors.length; i++) {
+			item.creators.push(ZU.cleanAuthor(editors[i], 'editor', false));
+		}
+	} else {
+		//e.g. Münchener Kommentar zum BGB
+		//from https://beck-online.beck.de/?vpath=bibdata%2fkomm%2fmuekobgb_7_band2%2fbgb%2fcont%2fmuekobgb.bgb.p305.htm
+		item.publicationTitle = ZU.trimInternal(citationFirst);
+	}
+	
+	var editionText = ZU.xpathText(doc, '//div[@class="dk2"]//span[@class="citation"]/text()[preceding-sibling::br]');
+	if (editionText) {
+		item.edition = editionText.match(/\d+/)[0];
```

Comment:
we're positive that edition always has a number? Couldn't be "Komplett revidierte Neuauflage" or something like that?